src/components/redux/reducers/index.jsx

```diff
@@ -1,16 +1,53 @@
 import { combineReducers } from "redux";
+import {
+  ADD_TO_CART,
+  DELETE_FROM_CART,
+  MODIFY_QUANTITY,
+  SUM_TOTAL_PRICE,
+  ADD_ONE_MORE_ITEM,
+  REMOVE_ONE_MORE_ITEM,
+  QUANTITY_IN_SINGLE_PRODUCT,
+} from "../actions/actionsTypes";
 
-const currentBoardIDReducer = (state = [], action) => {
+const addOrDeleteProductData = (state = [], action) => {
   switch (action.type) {
-    case "CURRENT_BOARD_ID":
+    case ADD_TO_CART:
+      return [...state, action.payload];
+    case DELETE_FROM_CART:
+      return action.payload;
```

Comment:
shouldn't this delete the payload from the existing state?

Reply:
We are deleting one item from our Redux state and then dispatching the new modified data